watson/cli.py

```diff
@@ -452,46 +457,62 @@ def report(watson, current, from_, to, projects,
 
     if format_json:
         click.echo(json.dumps(report, indent=4, sort_keys=True))
+        return
+
+    lines = []
+    # use the pager, or print directly to the terminal
+    if pager or (pager is None and
+            watson.config.getboolean('options', 'pager')):
+        use_pager = True
+        def _print(line):
+            lines.append(line)
     else:
-        click.echo('{} -> {}\n'.format(
-            style('date', '{:ddd DD MMMM YYYY}'.format(
-                arrow.get(report['timespan']['from'])
+        use_pager = False
+        def _print(line):
```

Comment:
There should be a blank line before inline function definitions (see also [Travis log](https://travis-ci.org/TailorDev/Watson/builds/315085764?utm_source=github_status&utm_medium=notification)).

Reply:
fixed!